docs/source/usecases/customrule.rst

```diff
@@ -0,0 +1,52 @@
+Config - Custom Rules
```

Comment:
Instead of calling them Custom rules, let's Call out as # AWS Config Rule Support. Custom rules is a ambiguous and can reference actually other aws features (cwe also uses rule nomenclature).

also this needs to be linked or inline on the page with the other lambda modes
http://www.capitalone.io/cloud-custodian/docs/policy/lambda.html